xamarin/Xamarin.Forms
# [iOS] Fix non visible TextColor issue in Entry using Visual

### Description of Change ###

The main issue is solved in native Material Components iOS library:
https://github.com/material-components/material-components-ios/issues/10081

We did a color workaround https://github.com/xamarin/Xamarin.Forms/issues/12246 to avoid the issue but we will investigate in release new binding for Material Components library. This PR just include a small change in the color workaround to fix the entry text no visible issue using the default color for now.

The default color alpha is -1, and that's why we set the default alpha to zero in the color workaround.

### Issues Resolved ### 

- fixes #12809

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
![fix12809](https://user-images.githubusercontent.com/6755973/101149793-b300eb00-361f-11eb-9234-f0ea46418f5b.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12809 on iOS simulator or device. Write in each Entry and verify that the text color is always visible.

### PR Checklist ###
